common/queries/servers/server_insert.php

```diff
@@ -1,25 +1,19 @@
 <?php
 
-function server_insert($pdo, $server_name, $address, $port, $expire_timestamp, $salt, $guild_id)
+function server_insert($pdo, $expire_time, $server_name, $address, $port, $guild_id)
 {
-    $expire_date = date("Y-m-d H:i:s", $expire_timestamp);
     $stmt = $pdo->prepare('
         INSERT INTO servers
-           SET server_name = :server_name,
+           SET expire_date = FROM_UNIXTIME(:expire_time),
```

Comment:
FROM_UNIXTIME doesn't seem to work here, for some reason. My small fix to get server rentals working was to add the `$expire_date = date("Y-m-d H:i:s", $expire_timestamp);` line that manually makes the timestamp -> date conversion before passing the value to the db

Reply:
I don't think `FROM_UNIXTIME` was the issue; I believe it was working fine when we converted the queries to PDO. Maybe try it first without merging into the repo? If it works, then it can be merged.

Reply:
@jacob-grahn ^^